TESTS/usb_device/basic/USBTester.cpp

```diff
@@ -60,7 +59,11 @@ USBTester::USBTester(USBPhy *phy, uint16_t vendor_id, uint16_t product_id, uint1
     int_in = resolver.endpoint_in(USB_EP_TYPE_INT, 64);
     int_out = resolver.endpoint_out(USB_EP_TYPE_INT, 64);
     MBED_ASSERT(resolver.valid());
+#if defined(MBED_CONF_RTOS_PRESENT)
     queue = mbed::mbed_highprio_event_queue();
+#else
```

Comment:
Is there any record in the logs of why it was using high priority specifically? I guess it's working as normal.

If there's no downside, I'd make it normal priority always. If there is a downside, put a comment about it here.